packages/components/src/components/post-card-control/post-card-control.scss

```
@@ -0,0 +1,177 @@
+@use '@swisspost/design-system-styles/core' as post;
+@use '@swisspost/design-system-styles/components/form-check';
+
+:host {
+  display: flex;
+  width: 100%;
+}
+
+:host-context(fieldset) {
+  &:host(:not(:last-child)) {
+    .card-control {
+      margin-bottom: post.$size-regular;
+    }
+  }
+}
+
+.card-control {
+  --post-card-control-border-color: #{post.$gray-60};
+  --post-card-control-bg: #{post.$white};
+  --post-card-control-color: #{post.$gray-80};
+  --post-card-control-input-border-color: #{post.$gray-80};
+  --post-card-control-input-bg: #{post.$white};
+
+  flex-basis: 100%;
+  display: flex;
+  gap: 0 post.$size-mini;
+  padding: post.$size-regular;
+  background-color: var(--post-card-control-bg);
+  border: post.$size-line solid var(--post-card-control-border-color);
+  border-radius: post.$border-radius;
+  color: var(--post-card-control-color);
+
+  cursor: pointer;
+  transition: background-color 100ms linear, border-color 100ms linear;
+
+  .card-control--input {
+    flex: 0 0 auto;
+    margin: post.$size-micro 0;
+    background-color: var(--post-card-control-input-bg);
+    border-color: var(--post-card-control-input-border-color) !important;
+    color: var(--post-card-control-input-border-color) !important;
+    cursor: inherit;
+    transition: border-color 100ms ease-in-out;
+
+    &:focus,
+    &:focus-visible {
+      box-shadow: none;
+    }
+
+    ~ .card-control--label {
+      flex-grow: 2;
+      margin: post.$size-micro 0;
+      color: inherit !important;
+      pointer-events: none;
+    }
+  }
+
+  .card-control--description {
+    font-size: 0.75rem;
+  }
+
+  .card-control--icon {
+    flex: 0 0 auto;
+    width: post.$size-big;
+    height: post.$size-big;
+    pointer-events: none;
+
+    > slot {
+      > * {
+        width: 100%;
+        height: 100%;
+      }
+    }
+  }
+
+  &:not(.is-disabled) {
+    // order matters!
```

```suggestion
    // order matters! because...
```
Adding more details could be better if we need to refactor one day.